bip-0099.mediawiki

```diff
@@ -0,0 +1,333 @@
+paths often leads to misunderstandings and unnecessarily delays the
+deployment of changes. 
+
+==Defintions==
+
+- Software fork: in free software, an existing project can be copied
+  and extended/maintained without permission from the original
+  project's maintainers.
+- Consensus fork: a divergence in the implementation of the
+  verification rules can impede the expected eventual convergence of
+  the network in a single chain that has the most proof of work and
+  also satisfies the rules. This can be intentional or be caused by a
+  bug in consensus validation reimplementations.
+- Softfork: an intentional consensus fork where everything that
+  was previously invalid remains invalid, but some things that were
+  valid also become invalid. Majority of miners can impose
+  this kind of changes and they also have some deployment advantages.
+- Hardfork: an intentional consensus fork that is not a softfork.
+
+==Taxonomy of consensus forks==
+
+===Accidental consensus fork===
+
+Software forks are very different in nature from consensus rules forks. No software
+maintainer has special powers over consensus rules changes. There's
+many good reasons (experimentation, lack of features, independent
+development, diversity, etc) to fork the Bitcoin core software and it's good
+that there's many alternative implementations of the protocol (forks
+of Bitcoin core or written from scratch).
+
+But sometimes a bug in the reimplementaion of the consensus
+validation rules can prevent alternative implementation users from
+following the longest (most work) valid chain. This can result in
+those users losing coins or being defrauded, making reimplementations
+of the consensus validation rules very risky. Note that a natural
+language specification of those rules doesn't help since the
+consensus is not determined by such specification but by the software
+that the majority of the network runs. That's why "the implementation
+is the specification".
+
+But Bitcoin core contains many more things than just consensus
+validation and it would be unreasonable for all alternative
+implementations to depend on it. Bitcoin core should not be the
+specification. That's why the consensus validation is being separated
+into a libbitcoinconsensus library with a C API easily accessible from
+any language. This makes alternative implementations much more secure
+without burdening them with specific design choices made by Bitcoin
+core. It is to be noted that sharing the same code for consensus
+validation doesn't prevent alternative implementations from
+independently changing their consensus rules: they can always fork
+the libbitcoinconsensus project (once it is in a separate repository). 
+
+Hopefully libbitcoinconsensus will remove this type of consensus fork
+which - being accidental - obviously don't need a deployment plan.
+
+====11/12 March 2013 Chain Fork====
+
+There is a precedent of an accidental consensus fork at height 225430. 
```

Comment:
There's a post-mortem about the March 2013 fork in [bip50](https://github.com/bitcoin/bips/blob/master/bip-0050.mediawiki), perhaps that should be linked or worked into this section?

https://github.com/bitcoin/bips/blob/master/bip-0050.mediawiki